### ISSUE TITLE
Add 'related' property to IIIF Manifests

### ISSUE THREAD

**Issue by mikeapp** [yalelibrary/YUL-DC#1374]
**Story**

The IIIF Manifest property `related` can contain a link to a related web page.  In our case, we'd like to provide a link to the catalog page.   For the spec, see IIIF Presentation API: https://iiif.io/api/presentation/2.1/#related

Example:
```
 "related":{
    "@id": "https://collections.library.yale.edu/catalog/2004518",
    "format": "text/html",
    "label": "Yale Digital Collections page"
  }
```

**Acceptance**
- [ ] IIIF Manifests are created with the `related` property

**Comment by FCRodriguez7** [comment]
PR: https://github.com/yalelibrary/yul-dc-management/pull/673

**Comment by FCRodriguez7** [comment]
Released management v2.45.2
